README.md

```diff
@@ -123,6 +123,54 @@ If you want to use Isaac Gym, use python3.8 (not python3.9)
 5. nvidia-smi` to check that you have NVIDIA drivers and proper cuda installations.
 6. (Assuming you already have mlflow running in a different terminal), Run `python -m main services/environment=ant`
 
+## Petting Zoo
```

Comment:
This full section should be merged in with the other instructions (basically you need to add the multi-agent ALE stuffs for Macs.

Reply:
The installation instruction is merged to Section `Getting Started` while examples are merged to Section `Configuration`. MAC installation and licence activation are moved to `doc/development_setup.md`